plugins/modules/aci_rest.py

```diff
@@ -329,6 +329,8 @@ def main():
         method=dict(type='str', default='get', choices=['delete', 'get', 'post'], aliases=['action']),
         src=dict(type='path', aliases=['config_file']),
         content=dict(type='raw'),
+        output_path=dict(type='str'),
```

Comment:
It seems that output_path is already in the aci_argument_spec().
And you may want to add test task for this update.

Reply:
Sorry, missed this.
Yes, I'll have to make that change to the module.